apps/src/sites/studio/pages/code-studio.js

```diff
@@ -10,6 +10,9 @@ import $ from 'jquery';
 
 import { getStore } from '@cdo/apps/code-studio/redux';
 import { setRtlFromDOM } from '@cdo/apps/code-studio/isRtlRedux';
+import { setUserSignedIn } from '@cdo/apps/code-studio/progressRedux';
```

Comment:
One other thing to note is that at this point signInState probably deserves its own reducer - progress doesn't really seem like the right place for it. I'm not going to make that change at this point though.